runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md

```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way, and it overwrites its operands when there is an unexpected change on them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
```

Comment:
Hyperconverged Cluster Operator configures clusters in an opinionated **way and overwrites** its operands when there is an unexpected change **to** them.

Reply:
done